liquibase-standard/src/main/java/liquibase/executor/jvm/JdbcExecutor.java

```diff
@@ -180,6 +177,13 @@ public void execute(final SqlStatement sql, final List<SqlVisitor> sqlVisitors)
         execute(new ExecuteStatementCallback(sql, sqlVisitors), sqlVisitors);
     }
 
+    private void setParameters(final PreparedStatement pstmt, final RawParameterizedSqlStatement sql) throws SQLException {
+        final List<?> parameters = sql.getParameters();
+        for (int i = 0; i < parameters.size(); i++) {
+            pstmt.setObject(i + 1, parameters.get(i));
```

Comment:
whow, PreparedStatement index starts at 1. Good catch!

Reply:
Thanks. Any suggestions or feedback to help get this merged?